tests/unit/test_logging.py

```diff
@@ -0,0 +1,42 @@
+import logging
+import os
+import time
+
+from pip._internal.utils.logging import IndentingFormatter
+
+
+class Test_IndentingFormatter(object):
+    """
+    Test `pip._internal.utils.logging.IndentingFormatter`
+    """
+
+    def setup(self):
+        self.old_tz = getattr(os.environ, 'TZ', None)
+        os.environ['TZ'] = 'UTC'
+        if 'tzset' in dir(time):
```

Comment:
I would include the same comment about why the "if" check is needed since this is a different file.